compiler/src/parsing/parser.dyp

```diff
@@ -409,7 +409,7 @@ lam_args :
   | patterns? { Option.value ~default:[] $1 }
 
 lam_expr :
-  | lparen lam_args trailing_comma? rparen thickarrow block_or_expr { Exp.lambda ~loc:(symbol_rloc dyp) $2 $6 }
+  | lparen lam_args rparen thickarrow block_or_expr { Exp.lambda ~loc:(symbol_rloc dyp) $2 $5 }
```

Comment:
Not a problem, just curious why this change is needed?

Reply:
The trailing comma is included in the patterns directly now! (There's a test for this, and it still passes 👍🏽)